mpc:
  dt: 0.1
  predict_steps: 10
  max_steer: 0.785
  max_dsteer: 1.0       #0.6
  max_speed: 2.0
  min_speed: -2.0
  max_accel: 5
  wheel_base: 0.6
  delay_num: 0
  test_mpc: false
  matrix_q: [100.0, 100.0, 10.0, 20.0]
  # matrix_q: [100.0, 100.0, 5.0, 5.0, 2.0]
  #pₓ  py v steer θ 
  matrix_r: [0.1, 0.01]
  # matrix_rd: [0.01, 0.01]
  matrix_rd: [5.0, 10.0]

  # matrix_q: [100.0, 100.0, 2.0]
  # matrix_r: [0.1, 0.01]
  # matrix_rd: [0.01, 10.0]
